### PR TITLE
fix route forwarding despite invalid token

### LINKED ISSUE #144: Confirmation Code page accepts any input
Currently, the user is redirected to the Sign-In page regardless of what they enter as the confirmation code. This can be misleading as redirecting to the sign-in page might indicate that the email has been verified.
We can improve the Confirmation code page by displaying an error if the user does not input the correct validation. 

P.S Should we also try to show a success/verified message if the correct code is entered?

### PR DESCRIPTION
**Pull request by codingfabi**
Closes #144

### CODE REVIEW

File: frontend/pages/confirm-email.tsx [hunk 1]
```diff
@@ -34,7 +36,19 @@ function confirmEmail(){
...
-  const [verifyAccount] = useMutation(VERIFY_ACCOUNT)
+  const [errorState, setErrorState] = useState(false)
+
+  const [verifyAccount] = useMutation(VERIFY_ACCOUNT,{
+    onCompleted: (result) => {
+      if(result.verifyAccount.success === true){
+        setErrorState(false)
+        router.push('/sign-in')
+      }
+      if(result.verifyAccount.success === false){
```
Comment (sami1riaz) [inline frontend/pages/confirm-email.tsx:47]: This should be an else rather than an if statement

File: frontend/pages/confirm-email.tsx [hunk 2]
```diff
@@ -85,6 +98,13 @@ function confirmEmail(){
         Confirm eMail Address
```
Comment (sami1riaz) [inline frontend/pages/confirm-email.tsx:98]: Minor, but eMail should be Email

File: frontend/pages/confirm-email.tsx [hunk 3]
```diff
@@ -85,6 +98,13 @@ function confirmEmail(){
...
       </Button>
       </form>
+      <Typography variant="body2" align="center">
+            <div style={{marginTop: 10}}>
+              { errorState && (
+                <Alert severity="error">The token you try to enter is invalid.</Alert>
```
Comment (sami1riaz) [inline frontend/pages/confirm-email.tsx:104]: try should be tried here.

**Review by sami1riaz — APPROVED** [review]
This can be merged once the minor comments are dealt with

**Comment by codingfabi** [comment]
addressed minor comments, merging this pr now

**Pull request merged**